src/utilities/signtx.ts

```diff
@@ -0,0 +1,67 @@
+import WalletStore from "@/store/WalletStore";
+import { MyAlgoWalletSession, WebMode } from "@algo-builder/web";
+import {
+	JsonPayload,
+	WalletMultisigMetadata,
+} from "@algo-builder/web/build/algo-signer-types";
+import algosdk, { encodeObj } from "algosdk";
+import { toRaw } from "vue";
+import { convertBase64ToUnit8Array, convertToBase64 } from "./convert";
+
+export async function signMultisigUsingMyAlgoWallet(
+	txnBase64: string,
+	txn: algosdk.Transaction
```

Comment:
why do we need both? base64 & the raw txn object? Can't we just apply an exisiting algosdk api on the txn object to get base64 version?

Reply:
the fn is called in two places one returns the encoded txn type and another transaction type
so that's why to make it clearly understandable that is expects transaction type, we have both the params